frontend/Screens/Sightings/SightingViewScreen.js

```diff
@@ -106,21 +111,30 @@ export default function SightingViewScreen({ navigation }) {
                 <RefreshControl refreshing={refreshing} onRefresh={onRefresh} />
             }
         >
-            <View style={{ justifyContent: 'space-evenly' }}>
-                {/* TODO: Replace sightingId with the number sighting for the specific turtle. */}
-                {/* <TurtleText titleText={`Sighting #${sightingId}`} /> */}
-                <TurtleText titleText={`Turtle #${turtleNumber}`} />
-                <TurtleText titleText="Mark: " baseText={mark} />
-                <TurtleText titleText="Date: " baseText={moment(date).format('l')} />
-                <TurtleText titleText="Length: " baseText={`${length} mm`} />
-                <TurtleText titleText="Location: " baseText={location} />
-            </View>
-            {/* map */}
-            <View style={{ width: '100%', height: 200 }}>
-                <TurtleMapView markers={markerList} />
-            </View>
-            <Gallery images={images} />
-            <TurtleText titleText="Notes: " baseText={notes} />
+            {
+                loading && !refreshing &&
+                <LoadingSpinner animating={loading}/>
+            }
```

Comment:
Only render the LoadingSpinner when the screen is in a loading state and not a refreshing state. This prevents there from being space on the screen when nothing is being loaded and  prevents there from being both a refreshing and a loading icon at the same time